azure-functions/acbs-function/mappings/facility/facility-guarantee-amend.js

```diff
@@ -0,0 +1,26 @@
+const facilityGuaranteeAmend = (amendment) => {
+  try {
+    const { amount, facilityGuaranteeDates } = amendment;
+    const record = {};
+
+    if (!amount && !facilityGuaranteeDates) {
+      throw new Error('Either amount or facilityGuaranteeDates.guaranteeExpiryDate is required.');
```

Comment:
Should the condition also check for `guaranteeExpiryDate`? As the error message mentions this, but it's currently only checking for `facilityGuaranteeDates`